Makefile

```diff
@@ -84,21 +84,29 @@ tools_setup:
 	@echo "setup tools"
 	@cd tools && make
 
+install_test_python_dep:
+	@echo "install python requirments for test"
```

Comment:
seems not need this cmd🤔

Reply:
em... strange, I didn’t change this part